pkg/registry/core/serviceaccount/storage/token.go

```diff
@@ -30,6 +30,7 @@ import (
 	"k8s.io/apiserver/pkg/authentication/authenticator"
 	genericapirequest "k8s.io/apiserver/pkg/endpoints/request"
 	"k8s.io/apiserver/pkg/registry/rest"
+
```

Comment:
nit: unrelated formatting change

Reply:
I won't write any more of these, the few I did was to attempt to underscore the need to remain mindful when composing large changesets such as this. I think if we want to provide meaningful oversight to one another it is important to simplify review as much as possible.